agentops_dcp.go

```diff
@@ -50,6 +66,117 @@ type VbSeqNoEntry struct {
 // GetVBucketSeqnosCallback is invoked with the results of `GetVBucketSeqnos` operations.
 type GetVBucketSeqnosCallback func([]VbSeqNoEntry, error)
 
+// OpenStream opens a DCP stream for a particular VBucket.
+func (agent *Agent) OpenCollectionStream(vbId uint16, flags DcpStreamAddFlag, vbUuid VbUuid, startSeqNo,
```

Comment:
I've added this as a new function so that we can maintain compatibility with older clients. If we bump gocbcore to 7.2/8? then we can probably drop the other function?